docs/UserGuide.md

```diff
@@ -379,9 +379,10 @@ Examples:
 
 Undoes the most recent undoable command. Undoable commands include: add, clone, delete, clear, edit.
 
-Format: `undo`
+#### Format: 
+`undo`
 
-Examples:
+#### Example commands:
 * `add n/John Doe p/98765432 e/johnd@example.com o/Barber a/John Street, Block 123, #01-01` followed by `undo` will
   delete the added contact.
 * `clone 1`, assuming there is a contact to clone, followed by `undo` will delete the cloned contact.
```

Comment:
```suggestion
* `clone 1`, assuming there is a contact to clone, followed by `undo` will delete the cloned contact
```

similar change based on the comment above

Reply:
Yep, will rectify the fullstops! Thanks!